src/core/utils/terrain/world_settings.js

```diff
@@ -1,10 +1,13 @@
-import { BlockType } from '@aresrpg/aresrpg-world'
+import { BlockType, WorldConf } from '@aresrpg/aresrpg-world'
 
-// constants
-export const world_patch_size = Math.pow(2, 6)
-// max cache radius as a power of two
-export const world_cache_pow_limit = 4 // 4 => 16 patches radius
+// World static config
+WorldConf.patchPowSize = 6 // as a power of two (6 => 64 blocks)
+// max cache radius
+WorldConf.cachePowLimit = 2 // as a power of two (4 => 16 patches radius)
+// debug vars
+WorldConf.debug.patch.borderHighlightColor = BlockType.NONE // BlockType.DBG_LIGHT
 
+// TODO remove hardcoding and retrieve dynamic value from world
```

Comment:
```suggestion
// TODO: remove hardcoding and retrieve dynamic value from world
```

https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight